src/app/inzage/brpApi.ts

```diff
@@ -0,0 +1,31 @@
+import { ApiClient } from '@gemeentenijmegen/apiclient';
```

Comment:
Deze file is dubbelop

Reply:
klopt, inzageproject is 1-op-1 geïmporteerd. Issue van maken.